letsdrill.rb

```diff
@@ -1,15 +1,29 @@
 def get_letter_grade(integer)
  
   #Put your code here!
-
+  if (integer >= 90)
+    "A"
+  elsif (integer >= 80)
+    "B"
+  elsif (integer >= 70)
+    "C"
+  elsif (integer >= 60)
+    "D"
+  else
+    "F"
+  end
```

Comment:
Nice! Note that parentheses aren't required for if/else statements, and that a case statement could also work here.